src/components/Header.js

```diff
@@ -67,22 +67,13 @@ const Header = ({ setContext }) => {
     element: Link,
   }));
 
-  const usingSparkExamples = useUsingSparkData().examples.map((page) => ({
```

Comment:
Why'd we remove this?

Reply:
I initially removed it because the folder is empty. I took another look and saw that we're pulling this folder and also a Guides folder in the widescreen navigation, so I added both of those to the narrow nav as well. The goal is to have both navs be in the same order.